docs/dev-manuals/tooling/techstack-and-service-creation.md

```diff
@@ -16,15 +16,15 @@ A user client sends a request to Nginx to retrieve the frontend. The frontend th
 
 Dapr acts as a middleware abstracting communication between services. This enables metric and trace collection via Open Telemetry. Services communicate via Dapr sidecars. Each sidecar is associated with one service and known by the Dapr placement / master service, also known as Dapr Operator Service. The Dapr Operator feeds the Open Telemetry collector with metrics and traces.
 
-A full Zipkin stack stores and displays traces, while metrics are collected by Grafana / Prometheus. Each Service involved in GraphQL queries needs to expose a GraphQL compatible API. Each services needs to be registered in the GraphQL gateway.
+A full Zipkin stack stores and displays traces, while metrics are collected by Grafana / Prometheus. Each Service involved in GraphQL queries needs to expose a GraphQL Federation 2 compatible API. Services involved in GraphQL queries need to be registered in the GraphQL gateway.
 
 ### Infastructure (as Code)
 
 The current plan is to support two deployments:
 
 1. Kubernetes
 
-   - State-of-the-art container orchestration. Widely used for microservices in science and the industry. Powerful but tedious to set up. Supports multi-node deployments, key stores, load balancing, auto scaling, etc.
+   - State-of-the-art container orchestration. Widely used for microservices in science and the industry. Powerful but tedious to set up (without infrastructure as code tools). Supports multi-node deployments, key stores, load balancing, auto scaling, etc.
```

Comment:
but we use infrastructure as code tools
idk, mentioning sth is tedious without what is basically the title of the section is a bit strange to me
but idk

Reply:
Ok, i have removed that